sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventDataBatch.java

```diff
@@ -7,8 +7,8 @@
 import com.azure.core.amqp.exception.AmqpException;
 import com.azure.core.amqp.exception.ErrorCondition;
 import com.azure.core.util.logging.ClientLogger;
-import com.azure.messaging.eventhubs.implementation.AmqpConstants;
-import com.azure.messaging.eventhubs.implementation.ErrorContextProvider;
+import com.azure.core.amqp.implementation.AmqpConstants;
```

Comment:
With modules in Java 9, we should figure out if we really need to access implementation packages from other modules.

Reply:
Unfortunately this is needed because we didn't want to expose proton-j reactor APIs in the public API.